internal/command/jsonformat/plan.go

```diff
@@ -106,6 +107,18 @@ func (plan Plan) renderHuman(renderer Renderer, mode plans.Mode, opts ...plans.Q
 				renderer.Colorize.Color("\n[reset][bold][red]Planning failed.[reset][bold] Terraform encountered an error while generating this plan.[reset]\n\n"),
 			)
 		} else {
+			// We had no changes, but deferred changes
+			if len(diffs.deferred) > 0 {
+				if haveRefreshChanges {
+					renderer.Streams.Print(format.HorizontalRule(renderer.Colorize, renderer.Streams.Stdout.Columns()))
+					renderer.Streams.Println("")
+				}
+				renderer.Streams.Print(
+					renderer.Colorize.Color("\n[reset][bold][green]No changes.[reset][bold] This plan requires another plan to be applied first.[reset]\n\n"),
+				)
+				return
```

Comment:
This is going to cause it not to render the deferred changes either I think?

Reply:
Yeah, I think so too 🤔 I wanted to get rid of some of the output below, maybe I'll just do a plain old else block then :)